marklogic-data-hub-central/ui/src/components/modeling/entity-type-modal/entity-type-modal.test.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import userEvent from "@testing-library/user-event";
+import EntityTypeModal from './entity-type-modal';
+
+describe('EntityTypeModal Component', () => {
+  test('Modal is not visible', () => {
+    const { queryByTestId } =  render(
+    <EntityTypeModal 
+      isVisible={false} 
+      toggleModal={jest.fn()}
+      newEntityAdded={jest.fn()}
+    />);
+
+    expect(queryByTestId('entity-type-modal')).toBeNull();
```

Comment:
This is about data-testid again. We seem to be using "Add Entity Type" text for assertion elsewhere. Since that works we can probably get rid of this data-testid? I will leave it up to you @brucean52.

Reply:
I've removed this testid and I'm doing queryByText for Null checks